.repos.yaml

```diff
@@ -0,0 +1,45 @@
+repositories:
```

Comment:
It would be great if you added instructions on using this to the README. Is it something like this:

`vcs import < Universal_Robots_ROS2_Driver/.repos.yaml`

Reply:
I think this is a good way to do it

Reply:
Done.